src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/help/Get-AzureRmRecoveryServicesBackupItem.md

```diff
@@ -116,6 +123,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Policy
```

Comment:
Please use the -UseFullTypeName flag and regenerate this file.  Also, you will need to move the ```yaml tag down to a new line for the -Policy parameter description (this is a known bug in yaml that will be fixed in a release coming soon).